src/simulation/CoreTests.cpp

```diff
@@ -88,7 +88,7 @@ TEST_CASE("3 nodes 2 running threshold 2", "[simulation][core3]")
         simulation->addPendingConnection(keys[0].getPublicKey(),
                                          keys[1].getPublicKey());
 
-        auto tBegin = std::chrono::system_clock::now();
+        // auto tBegin = std::chrono::system_clock::now();
```

Comment:
delete instead of commenting out

Reply:
The reason I went with commenting might not be obvious from the diff.

In the code, those variable are referenced in other [commented-out code](https://github.com/stellar/stellar-core/blob/aecb754bf4e17eb58545d85730ea1e5464104eda/src/simulation/CoreTests.cpp#L104). Just grep for `printStats` in this file.

I saw 3 approaches here:
- Uncomment the `printStats` call and use `#if defined(....)`
- Comment out the variables
- Remove the variables and the comment

Removing the variables and not the comments would be a bit odd methinks.
So, should I remove the `printStats` comments in addition to the `tBegin` variables ?

Reply:
yeah you can remove the commented out `printStats` linbes. Its looks like some debugging stuff that was left behind